operator/pkg/reconciliation/reconcile_endpoints.go

```diff
@@ -0,0 +1,107 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package reconciliation
+
+import (
+	"github.com/datastax/cass-operator/operator/internal/result"
+	api "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/datastax/cass-operator/operator/pkg/utils"
+)
+
+func (rc *ReconciliationContext) CreateEndpointsForAdditionalSeedService() result.ReconcileResult {
+	// unpacking
+	logger := rc.ReqLogger
+	client := rc.Client
+	endpoints := rc.Endpoints
+
+	logger.Info(
+		"Creating endpoints for additional seed service",
+		"endpointsNamespace", endpoints.Namespace,
+		"endpointsName", endpoints.Name)
+
+	if err := setOperatorProgressStatus(rc, api.ProgressUpdating); err != nil {
+		return result.Error(err)
+	}
+
+	if err := client.Create(rc.Ctx, endpoints); err != nil {
+		logger.Error(err, "Could not create endpoints for additional seed service")
+
+		return result.Error(err)
+	}
+
+	rc.Recorder.Eventf(rc.Datacenter, "Normal", "CreatedResource", "Created endpoints %s", endpoints.Name)
+
+	return result.Continue()
+}
+
+func (rc *ReconciliationContext) CheckAdditionalSeedEndpoints() result.ReconcileResult {
+	// unpacking
+	logger := rc.ReqLogger
+	dc := rc.Datacenter
+	client := rc.Client
+
+	logger.Info("reconcile_endpoints::CheckAdditionalSeedEndpoints")
+
+	desiredEndpoints := newEndpointsForAdditionalSeeds(dc)
```

Comment:
what do you think about adding a guard here that returns result.Continue() early if len(dc.Spec.AdditionalSeeds) == 0?